tests/conftest.py

```diff
@@ -81,9 +80,12 @@ def rsps():
 
 @pytest.fixture
 def disable_gzip():
-    os.environ["COGNITE_DISABLE_GZIP"] = "1"
-    yield
-    del os.environ["COGNITE_DISABLE_GZIP"]
+    try:
```

Comment:
try shouldn't be necessary, I believe. Pytest executes the teardown even when exceptions are raised

Reply:
Oh, thats cool, TIL 🤩